ApollosProject/apollos-apps
# Move features to ui connected

## DESCRIPTION

### What does this PR do, or why is it needed?
Moves content-single features to ui-connected package. Updates imports, naming, and adds a snapshot test. No other refactoring was done. 

### What design trade-offs/decisions were made?

Next steps for this component is to refactor the ui elements (TextFeature &
  ScriptureFeature).

### How do I test this PR?

Click on 'Spur One Another' article and verify all feature content still works as expected.
![image](https://user-images.githubusercontent.com/2528817/74683601-1d2ec900-518f-11ea-9974-fc4dd758b4a2.png)


### What automated tests did you write?
Yes, snapshot testing connected component.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
